shared/chat/inbox/index.native.js

```diff
@@ -85,7 +85,7 @@ class Inbox extends React.PureComponent<Props, State> {
   }
 
   componentWillReceiveProps(nextProps: Props) {
-    if (this.props.smallTeamsExpanded !== nextProps.smallTeamsExpanded && !nextProps.smallTeamsExpanded) {
+    if (this.props.smallTeamsHiddenCount === 0 && nextProps.smallTeamsHiddenCount > 0) {
```

Comment:
This is the intended effect, right?

Reply:
What is this suppose to do?

Reply:
I don't think so?

because you can scroll down if you have a lot of big teams (or channels). but you don't have many small teams.

You want to always scroll up since the list effectively changed. Otherwise you will just be in a broken state I think

Reply:
I meant that the old code had `this.boolVar != next.boolVar && !next.boolVar`, which is true only when `next.boolVar` is false and `this.boolVar` is true, i.e. `this.boolVar && !next.boolVar`.

Then I use that `smallTeamsExpanded` in the old code is equivalent to `smallTeamsHiddenCount === 0` in the new code.